### DISCUSSION TITLE
Update username

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1621] [Ideas]
It would be nice if users would be able to update their username from Monkeytype. Like update email.

**Comment by frogfrogfrogfrogfrog** [comment]
Yeah if you have a username that you don't like then being able to change it is a good idea.

It's not like usernames are concrete where you have to have a different username from everyone else

**Reply by Miodec** [reply]
> It's not like usernames are concrete where you have to have a different username from everyone else

Uh.. it does have to unique.

**Reply by unrealapex** [reply]
Yep.

**Reply by frogfrogfrogfrogfrog** [reply]
welp sorry for that mistake (for me whenever I typed in the first thing it was available so I made that assumption)

it'd still be a nice idea to be able to change your username
